lib/Optimizer/GraphOptimizer.cpp

```diff
@@ -2134,6 +2134,78 @@ static void optimizeQuantization(Function *F) {
   optimizeQuantizedMaxSplat(F);
 }
 
```

Comment:
Docs please.

Reply:
This was added by @rdzhabarov 9 month ago.

Reply:
hm :) what was added? (I thought `Docs please.` is for comment for the macro BOILER_PLATE_TEMPLATE)

Reply:
@ayermolo I believe @nadavrot still wanted some comments here on `FUNCTION_ENABLE_IF_TEMPLATE`

Reply:
@nadavrot Ooops, missed it. Would  you like me to add comment in separate pull request?